build.sbt

```diff
@@ -11,13 +11,13 @@ scalacOptions in (Compile, compile) ++= Seq(
   "-target:jvm-1.8",
   "-encoding", "UTF-8",
   "-feature",
-  "-Ywarn-unused",
-  "-Ywarn-unused-import",
-  "-Ywarn-infer-any"
+  "-Wunused:imports,locals",
+  "-Xlint:infer-any"
 )
 
 libraryDependencies ++= Seq(
-  "org.scalatest" %% "scalatest" % "3.0.1" % "test"
+  "org.scala-lang.modules" %% "scala-parallel-collections" % "1.0.3",
```

Comment:
This was added for the parallel safety test that uses `par`. In 2.13 this was moved to an external lib.